lib/diagnostic.rb

```diff
@@ -40,7 +40,9 @@
 def method_name
 end
 
-Diagnostic.defined_method = # call your method here
+Diagnostic.defined_method = def sum_times_two (a, b)
+                              2 * (a + b)
+                            end
 
```

Comment:
I see. I just realized I misread the question / maybe wasn't reading!

On Thu, Feb 11, 2016 at 10:19 AM, Jason Weeks <notifications@github.com>
wrote:

> In lib/diagnostic.rb
> <https://github.com/ga-wdi-boston/ruby-diagnostic/pull/43#discussion_r52614559>
> :
>
> > @@ -40,7 +40,9 @@
> >  def method_name
> >  end
> >
> > -Diagnostic.defined_method = # call your method here
> > +Diagnostic.defined_method = def sum_times_two (a, b)
> > +                              2 * (a + b)
> > +                            end
> >
>
> define then call
>
>
> def add_and_multiply(a, b)
>   2 * (a + b)
> end
>
> Diagnostic.defined_method = add_and_multiply(3, 4)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/ruby-diagnostic/pull/43/files#r52614559>
> .
>